docs/en/connector-v2/sink/RocketMQ.md

```diff
@@ -1,44 +1,43 @@
-| partition.key.fields | array   | no       | -                        |
-| format               | String  | no       | json                     |
-| field.delimiter      | String  | no       | ,                        |
-| common-options       | config  | no       | -                        |
+> Spark<br/>
+> Flink<br/>
+> SeaTunnel Zeta<br/>
 
-### topic [string]
+## Key features
 
-`RocketMQ topic` name.
+- [x] [exactly-once](../../concept/connector-v2-features.md)
 
-### name.srv.addr [string]
+By default, we will use 2pc to guarantee the message is sent to RocketMQ exactly once.
 
-`RocketMQ` name server cluster address.
+## Description
 
-### semantic [string]
+Write Rows to a Apache RocketMQ topic.
 
-Semantics that can be chosen EXACTLY_ONCE/AT_LEAST_ONCE/NON, default NON.
+## Sink Options
+
+|         Name         |  Type   | Required |         Default          |                                                                             Description                                                                             |
+|----------------------|---------|----------|--------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| topic                | string  | yes      | -                        | `RocketMQ topic` name.                                                                                                                                              |
+| name.srv.addr        | string  | yes      | -                        | `RocketMQ` name server cluster address.                                                                                                                             |
+| acl.enabled          | Boolean | no       | false                    | false                                                                                                                                                               |
+| access.key           | String  | no       |                          | When ACL_ENABLED is true, access key cannot be empty                                                                                                                |
+| secret.key           | String  | no       |                          | When ACL_ENABLED is true, secret key cannot be empty                                                                                                                |
+| producer.group       | String  | no       | SeaTunnel-producer-Group | SeaTunnel-producer-Group                                                                                                                                            |
+| semantic             | string  | no       | NON                      | Semantics that can be chosen EXACTLY_ONCE/AT_LEAST_ONCE/NON, default NON.                                                                                           |
```

Comment:
Do you read the `RocketMqSinkFactory` code? Some options I don't found in the code.

<img width="1227" alt="image" src="https://github.com/apache/seatunnel/assets/32193458/e13f99f7-8ce8-4c9a-84c0-ddd94c771146">

Reply:
Is this parameter `semantic`?